pkg/client/postgres/tasks.go

```diff
@@ -95,9 +97,26 @@ func (db *PostgresDB) ScheduleBackup(schedule string) error {
 	return db.client.Apply(db.Namespace, job)
 }
 
-func (db *PostgresDB) ListBackups() ([]string, error) {
-	var backups []string
-	return backups, nil
+type BackupItem struct {
+	URL          string
+	LastModified time.Time
+	Size         int64
+}
+
+func (db *PostgresDB) ListBackups() ([]*BackupItem, error) {
+	op := db.op.Configuration
+	doneCh := make(chan struct{})
```

Comment:
we dont really use channels for context anywhere else - Could we switch it to https://pkg.go.dev/github.com/minio/minio-go/v7#Client.ListObjects  and then pass the context through as a param ?

We probably also want to sort by LastModified desc

Reply:
At present it is using minio-go/v6 where ListObjects doesn't take context as param. ( https://pkg.go.dev/github.com/minio/minio-go/v6#Client.ListObjects ) . Should we upgrade to  minio-go/v7 ? 
I will add sort by LastModified desc